.ci/pytorch/test.sh

```diff
@@ -325,6 +325,10 @@ else
   DYNAMO_BENCHMARK_FLAGS+=(--device cuda)
 fi
 
+if [[ "${TEST_CONFIG}" == *multiprocess* ]]; then
+  DYNAMO_BENCHMARK_FLAGS+=(--multiprocess)
```

Comment:
I suppose this will only run the tests that support `--multiprocess`? I haven't looked into your previous change to the testing script.